### PR TITLE
Add support for unbundled openssl on AIX and other small fixes

### PR DESCRIPTION
**Pull request by groeges**
Signed-off-by: Steve Groeger <GROEGES@uk.ibm.com>

### CODE REVIEW

**Comment by pshipton** [comment]
test sanity osx

**Comment by pshipton** [comment]
jenkins test sanity osx

**Comment by pshipton** [comment]
jenkins compile aix

**Comment by pshipton** [comment]
AIX CryptoTest being enabled in eclipse/openj9#4922

**Comment by pshipton** [comment]
The osx failures are known problems unrelated to this change.

**Comment by pshipton** [comment]
AIX CryptoTest in eclipse/openj9#4922 succeeded

**Review by andrew-m-leonard — APPROVED** [review]

File: closed/src/java.base/aix/native/libjncrypto/NativeCrypto_md.c [hunk 1]
```diff
@@ -32,18 +32,22 @@
...
-    const char *libname = "libcrypto64.so.1.1";
-    const char *oldname = "libcrypto.so.1.0.0";
-    const char *symlink = "libcrypto.so";
+    const char *libname111 = "libcrypto.a(libcrypto64.so.1.1)";
+    const char *libname110 = "libcrypto.so.1.1";
+    const char *libname102 = "libcrypto.so.1.0.0";
+    const char *symlink = "libcrypto.a(libcrypto64.so)";
 
-    result = dlopen (libname, RTLD_NOW);
+    char *errorstr;
```
Comment (davehobbs) [inline closed/src/java.base/aix/native/libjncrypto/NativeCrypto_md.c:40]: It doesn't look like errorstr is needed.

**Pull request merged**